cumulusci/tasks/tests/test_util.py

```diff
@@ -111,6 +121,45 @@ def test_FindReplace(self, find_replace):
         task()
         find_replace.assert_called_once()
 
+    @mock.patch("cumulusci.tasks.util.find_replace")
+    def test_FindReplace_env_replace_error(self, find_replace):
+        os.unsetenv("bar")
+        with pytest.raises(TaskOptionsError):
+            task_config = TaskConfig(
+                {
+                    "options": {
+                        "find": "foo",
+                        "replace": "bar",
+                        "env_replace": True,
+                        "path": ".",
+                        "max": 1,
+                    }
+                }
+            )
+            task = util.FindReplace(self.project_config, task_config, self.org_config)
+            task()
+            find_replace.assert_called_once()
+
+    @mock.patch("cumulusci.tasks.util.find_replace")
+    def test_FindReplace_env_replace(self, find_replace):
+        os.environ["bar"] = "bars"
```

Comment:
Please use mock.patch.dict to make sure we only add this to the environment temporarily. We don't want to let it leak into other tests.

I see you're cleaning it up on line 161, but that won't run if the test fails before then. Using mock.patch.dict is the best way to make sure that happens. If you search for mock.patch.dict I think you'll find some other examples where we do this.